### ISSUE TITLE
Suggestion: using EvaluationDose

### ISSUE THREAD

**Issue by Kiragroh** [brjdenis/VarianESAPI-EQD2Converter#1]
Hi,

very nice and good to understand script.

I was wondering why you use DICOM for the conversion. If you use:

ExternalPlanSetup plan = course.AddExternalPlanSetup(ss);
EvaluationDose eDose = plan.CreateEvaluationDose();

You can modify the DoseVoxels with ESAPIs SetVoxel(slice, doseplane [,]) function

Hopefully this is helpful.

**Comment by brjdenis** [comment]
@Kiragroh Wow, for some reason github didn't tell me about this comment. 

Anyway, thanks for the comment! I was sure you can't do that, but it looks like i was wrong. I will fix this, most definitely! 
Thanks!

**Comment by brjdenis** [comment]
@Kiragroh it worked! Thank you very much for the suggestion.

**Comment by Kiragroh** [comment]
I Know.^^ I implemented it the day after my suggestion. Very nice. Two other suggestions:
- in your code all DVH for all selected structures will be calculated again when a new structure is selected. This makes the program very slow after selecting one big structure. I changed some things in the RemoveDVH class and now it is fine
- I programmed to autoselect the structures with alphaBeta value in the preview window.
